tests/e2e/chains/ethereum/Dockerfile.npm

```diff
@@ -0,0 +1,5 @@
+FROM node:18
```

Comment:
Even Node.js 20.x is in Maintenance. Should we use node:22?
https://nodejs.org/en/about/previous-releases

Reply:
It might be better to make a separate PR for updating the node version.

Reply:
OK.